things/templates/things/index.html

```diff
@@ -2,9 +2,11 @@
 
 {% block header %}Thing Review{% endblock %}
 {% block body %}
-    <p>Welcome to thing review. Here you can give your opinion on things. (review them)</p>
+    <p>Welcome to <i>thing</i> review. Here you can give your opinion on <i>things.</i> (review them)</p>
```

Comment:
There is inconsistent placement of `<i>` tags in regard to punctuation.
```html
<i>thing</i>.
```
:arrow_up: is the preferred method.
```suggestion
    <p>Welcome to <i>thing</i> review. Here you can give your opinion on <i>things</i>. (review them)</p>
```